src/main/java/org/engine/process/performance/multi/EngingPerformanceMultiPeriods.java

```diff
@@ -179,11 +207,88 @@ public String getOutput() {
 		output.append("The standard deviation  of total is "+utils.standardDeviation(totalDiffTimeArray)).append(endl);
 		
 		output.append("Export to CSV ").append(endl);
+		output.append("NUM_OF_INTERCAES").append(seperator).append(numOfInteraces).append(endl);
+		output.append("NUM_OF_UPDATES").append(seperator).append(numOfUpdates).append(endl);
 		output.append("CREATE").append(endl);
-		output.append(utils.createCsvFile(rowDataToSourceDiffTimeCreateArray,sourceToUpdateDiffTimeCreateArray,totalDiffTimeCreateArray,sourceName)).append(endl);
+		output.append(utils.createCsvFileDataInRows(rowDataToSourceDiffTimeCreateArray,sourceToUpdateDiffTimeCreateArray,totalDiffTimeCreateArray,sourceName)).append(endl);
 		output.append("UPDATE").append(endl);
-		output.append(utils.createCsvFile(rowDataToSourceDiffTimeUpdateArray,sourceToUpdateDiffTimeUpdateArray,totalDiffTimeUpdateArray,sourceName)).append(endl);
-			
+		output.append(utils.createCsvFileDataInRows(rowDataToSourceDiffTimeUpdateArray,sourceToUpdateDiffTimeUpdateArray,totalDiffTimeUpdateArray,sourceName)).append(endl).append(endl);
+		
+		
+		
+		output.append("NUM_OF_INTERCAES").append(seperator).append(numOfInteraces).append(endl);
+		output.append("NUM_OF_UPDATES").append(seperator).append(numOfUpdates).append(endl);
+		if( durationInMin > 0 ) {
+			output.append("DURATION(MIN)").append(seperator).append(num_of_cycles).append(endl);
+		}
+		else {
+			output.append("NUM_OF_CYCLES").append(seperator).append(num_of_cycles).append(endl);
+		}
+		output.append("INTERVAL").append(seperator).append(interval).append(endl); 
+		output.append(utils.createCsvFileDataInColumns(rowDataToSourceDiffTimeCreateArray,sourceToUpdateDiffTimeCreateArray,totalDiffTimeCreateArray,
+							rowDataToSourceDiffTimeUpdateArray,sourceToUpdateDiffTimeUpdateArray,totalDiffTimeUpdateArray,sourceName)).append(endl);
+				                       
+		
+		
 		return output.toString();
 	} 
-}
+	
+	private SingleCycle runSingleCycle(int numOfCycle) throws IOException, RestClientException, InterruptedException {
+		
+		String externalSystemID = utils.randomExternalSystemID();
+		SingleCycle singleCycle = new SingleCycle(); 
+		double lat = 4.3;
+		double longX = 6.4;
+
+		for(int j = 0 ; j < num_of_updates_per_cycle; j++) {
+			System.out.println("UPDATE " + j);
```

Comment:
Change all System.out.println to log4j messages according to the appropriate level